src/geo/transform.js

```diff
@@ -559,6 +559,13 @@ class Transform {
         this._posMatrixCache = {};
         this._alignedPosMatrixCache = {};
     }
+
+    getPitchScaleFactor() {
```

Comment:
Maybe we could change the name to something like `maxPitchScaleFactor` along with a comment something along the lines of "returns the pitch scaling factor for the very top of the viewport"?